SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/LoadBalancingEditorExtension/SpatialGDKEditorLBExtensionTest.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "SpatialGDKEditorModule.h"
+#include "SpatialGDKEditorSettings.h"
+
+#include "TestLoadBalancingStrategyEditorExtension.h"
+
+#define LB_EXTENSION_TEST(TestName) \
+	GDK_TEST(SpatialGDKEditor, LoadBalancingEditorExtension, TestName)
+
+namespace
+{
+
+struct TestFixture
+{
+	TestFixture()
+		: ExtensionManager(FModuleManager::GetModuleChecked<FSpatialGDKEditorModule>("SpatialGDKEditor").GetLBStrategyExtensionManager())
+	{
+		// Make sure extension is not registered
+		ExtensionManager.UnregisterExtension<FTestLBStrategyEditorExtension>();
```

Comment:
This actually necessary? Is this state stored not entirely on `ExtensionManager`?

Reply:
It is entirely stored in the extension manager, I guess that's some remnants of clearing state before I added the fixture. Would be useful only if someone added another test without using the fixture, but I'm just being paranoid.